sdks/python/apache_beam/examples/inference/tfx_bsl/tensorflow_image_classification.py

```diff
@@ -0,0 +1,192 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+A pipeline that uses TFX RunInference API to perform image classification.
```

Comment:
A sample pipeline illustrating how to use Apache Beams run_inference api with tfx-bsl's CreateModelHandler API.

 ---- some more suggested text you might add---
The images for this example are taken from <insert something>.

Reply:
SG